src/platforms/windows/daemon/windowssplittunnel.h

```diff
@@ -0,0 +1,178 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef WINDOWSSPLITTUNNEL_H
+#define WINDOWSSPLITTUNNEL_H
+
+#include <QObject>
+#include <QString>
+#include <QStringList>
+
+// Note: include order matters here.
```

Comment:
I've noticed that including windows headers in our `classname.h` files tends to be playing with fire and leads to a lot of breakage if the include order isn't just right. Would it be possible to move this into `windowssplittunnel.cpp`?

Reply:
Or is that fixed by `WIN32_LEAN_AND_MEAN` this might actually be better...

Reply:
`WIN32_LEAN_AND_MEAN` makes windows.h just include the minimum set on headers, so far this has been making things a lot less "where did i import what, in which orders" , but still probably good idea to move that all into *.cpp :)